service-front/module/Application/view/application/pages/post_office/confirm_post_office.twig

```diff
@@ -51,8 +51,14 @@
                                 {{ id_method }}
                             </dd>
                             <dd class="govuk-summary-list__actions">
-                                <a class="govuk-link" href="post-office-documents">Change<span
-                                            class="govuk-visually-hidden"></span></a>
+                                <a
+                                    class="govuk-link"
+                                    href="post-office-documents"
+                                    aria-label="Change document"
+                                >
+                                    Change
+                                    <span class="govuk-visually-hidden"></span>
```

Comment:
This span should contain what the user is changing for accessibility needs where they can't see the link in context.